src/ui.rs

```diff
@@ -297,34 +314,111 @@ impl App {
                             self.return_to_channels();
                         }
                     }
+                    KeyCode::Char('r') => {
+                        // Quick reply to last message sender
+                        if let Some(ref last_sender) = self.last_message_sender.clone() {
+                            self.input_mode = InputMode::QuickReply {
+                                target_peer: last_sender.clone(),
+                            };
+                            self.add_to_log(format!("📤 Quick reply to {}", last_sender));
```

Comment:
These calls to add_to_log need to use Icons type to maintain windows compatability